traitsui/tests/test_labels.py

```diff
@@ -157,9 +158,8 @@ def _test_qt_labels_right_resizing(self, dialog_class):
 
         from pyface import qt
 
-        with store_exceptions_on_all_threads():
-            dialog = dialog_class()
-            ui = dialog.edit_traits()
+        with store_exceptions_on_all_threads(), \
+                create_ui(dialog_class()) as ui:
```

Comment:
There's one more test above (`test_qt_show_labels_right_without_colon`) that creates ui and doesn't dispose of it.

Reply:
Well spotted. I will do that here too.